probe_scraper/runner.py

```diff
@@ -89,6 +99,8 @@ def dump_json(data, file_name):
                         help='Directory to store output files in.',
                         action='store',
                         default='.')
+    parser.add_argument('--break-by-channel', action='store_true',
```

Comment:
One larger question here:
Why are we running the job two times?
Can we just output to per-channel and a monolithic file from one job run?